benches/polyhedron.rs

```diff
@@ -0,0 +1,110 @@
+#![feature(test)]
```

Comment:
need to add it to CI

Reply:
Already is, benches are run on nightly only, because the test feature is only available there